samples/sample_calibrate_eye_to_hand.py

```diff
@@ -0,0 +1,77 @@
+"""Hand-eye calibration sample."""
+import datetime
+
+import numpy as np
+import zivid.handeye
+
+
+def _acquire_checkerboard_frame(camera):
+    print("Capturing checkerboard image... ")
+    with camera.update_settings() as updater:
+        updater.settings.iris = 17
+        updater.settings.gain = 1.0
+        updater.settings.exposure_time = datetime.timedelta(microseconds=20000)
+        updater.settings.filters.gaussian.enabled = True
+    print("OK")
+    return camera.capture()
+
+
+def _enter_robot_pose(index):
+    inputted = input(
+        "Enter pose with id={} (a line with 16 space separated values describing 4x4 row-major matrix):".format(
+            index
+        )
+    )
+    elements = inputted.split(maxsplit=15)
+    data = np.array(elements, dtype=np.float64).reshape((4, 4)).transpose()
+    robot_pose = zivid.handeye.Pose(data)
+    print("The following pose was entered:\n{}".format(robot_pose))
+    return robot_pose
+
+
+def _main():
+    app = zivid.Application()
+    camera = app.connect_camera()
+
+    current_pose_id = 0
+    calibration_inputs = list()
+    calibrate = False
+
+    while not calibrate:
+        command = input(
+            "Enter command, p (to add robot pose) or c (to perform calibration):"
+        ).strip()
+        if command == "p":
+            try:
+                robot_pose = _enter_robot_pose(current_pose_id)
+
+                frame = _acquire_checkerboard_frame(camera)
+
+                print("Detecting checkerboard square centers... ")
+                result = zivid.handeye.detect_feature_points(frame.get_point_cloud())
+
+                if result:
+                    print("OK")
+                    res = zivid.handeye.CalibrationInput(robot_pose, result)
+                    calibration_inputs.append(res)
+                    current_pose_id += 1
+                else:
+                    print("FAILED")
+            except ValueError as ex:
+                print(ex)
+        elif command == "c":
+            calibrate = True
+        else:
+            print("Unknown command '{}'".format(command))
+
+    print("Performing hand-eye calibration...")
+    calibration_result = zivid.handeye.calibrate_eye_to_hand(calibration_inputs)
+    if calibration_result:
+        print("OK")
+        print("Result:\n{}".format(calibration_result))
```

Comment:
Converting calibration_result to string doesn't give expected behavior. The actual numbers for the residuals are missing, and we get stuff like this:
`
{ Hand-Eye transformation: [ [ 0.670959,  0.677084, -0.302277,  637.027719], 
  [ 0.735697, -0.557012,  0.385340, -598.974910], 
  [ 0.092536, -0.480932, -0.871861,  1159.374538], 
  [ 0.000000,  0.000000,  0.000000,  1.000000] ], Per-pose calibration residuals: { { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  }, { Residual for rotation in deg: , Residual for translation in mm:  } } }
`

Reply:
Also, trying to run `calibration_result.perPoseCalibrationResiduals()` gives
`TypeError: Unable to convert function return value to a Python type! The signature was
	(self: _zivid._zivid.handeye.CalibrationOutput) -> std::vector<Zivid::HandEye::CalibrationResidual, std::allocator<Zivid::HandEye::CalibrationResidual> >

Did you forget to `#include <pybind11/stl.h>`? Or <pybind11/complex.h>,
<pybind11/functional.h>, <pybind11/chrono.h>, etc. Some automatic
conversions are optional and require extra headers to be included
when compiling your pybind11 module.
`